.github/workflows/playground-blueprint.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Blueprint Blueprint with Repository and Branch
+
+on:
+  push:
+    branches:
+      - main
+      - playground-ready
+    tags:
+      - "**"
+
+jobs:
+  update-blueprint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
Latest version of actions/checkout@v4

Reply:
Fixed in [502ce31](https://github.com/aspirepress/AspireUpdate/pull/162/commits/502ce31973e241e0a067b8a2c4fef1572ca08ecc).